template/web/src/containers/ScrollyModuleContainer.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { FC, useEffect } from 'react'
+import Box from '@/atoms/Box'
+import { graphql } from 'gatsby'
+import { SanityScrollyModuleDefaultFields } from 'web/types/graphql-types'
+import gsap from 'gsap'
+import ScrollTrigger from 'gsap/ScrollTrigger'
+
+gsap.registerPlugin(ScrollTrigger)
```

Comment:
we should not register ScrollTrigger multiple times

Reply:
"There is no harm in registering the same plugin multiple times (but it doesn't help either)."